src/bika/lims/setuphandlers.py

```diff
@@ -515,19 +529,32 @@ def add_dexterity_setup_items(portal):
          "Interpretation Templates",
          "InterpretationTemplates")
     ]
+    setup = api.get_setup()
+    add_dexterity_items(setup, items)
+
+
+def add_dexterity_items(container, items):
+    """Adds a dexterity item, usually a folder in the container
+    :param container: container of the items to add
+    :param items: tuple of Id, Title, FTI
+    """
+    pt = api.get_tool("portal_types")
+    ti = pt.getTypeInfo(container)
+
+    # Temporary allow content type creation in content
+    ftis = map(lambda item: item[2], items)
+    temporary_allow_type(container, ftis)
```

Comment:
`temporary_allow_type` is a context manager, so you have to call it like you did below, e.g.:

`with temporary_allow_type(container, fti) as ct`

So I think you can delete these two lines